game/core/game_view.py

```diff
@@ -46,14 +46,15 @@ def __init__(
         arcade.resources.add_resource_handle("characters", "game/assets/characters")
         arcade.resources.add_resource_handle("maps", "game/assets/maps")
         arcade.resources.add_resource_handle("sounds", "game/assets/sounds")
+        arcade.resources.add_resource_handle("misc", "game/assets/misc")
```

Comment:
Could be `animation` to better match the new class:

```suggestion
        arcade.resources.add_resource_handle("animation", "game/assets/animation")
```

Then you could nest the sparkles like: `animation/sparkles/1.png`, etc.